inc/targetchange.class.php

```diff
@@ -0,0 +1,1502 @@
+<?php
+class PluginFormcreatorTargetChange extends CommonDBTM
+{
+
+   static function getEnumDestinationEntity() {
```

Comment:
I was planning this an an other PR with other code refactors, to make things easier to  review.